src/styles/index.scss

```
@@ -1,3 +1,121 @@
+$back-image: url(/src/images/star-active.svg);
+$button-color: #00acdc;
+$text-color: #060b35;
+$text-color-another: #616070;
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__image {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+  &__button {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    color: #fff;
+    text-transform: uppercase;
+    text-decoration: none;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+
+    width: 100%;
+    height: 40px;
+    box-sizing: border-box;
+    border: 1px solid $button-color;
+    border-radius: 5px;
+    background: $button-color;
+
+    &:hover {
+      background: #fff;
+      color: $button-color;
+    }
+  }
+}
+
+.product {
+  &__name {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+    color: $text-color;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    line-height: 14px;
+    text-align: left;
+    margin-bottom: 16px;
+    color: $text-color-another;
+  }
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    line-height: 18px;
+    margin-bottom: 16px;
+    color: $text-color-another;
+  }
+  &__price-value {
+    font-size: 16px;
+    font-weight: 700;
+    text-align: right;
+    color: #060b35;
+  }
+
+  &__price-header {
+    font-size: 12px;
+    text-align: left;
+    color: #616070;
+  }
```

Use scss nesting, fix it everywhere
```suggestion
  &__price {
    display: flex;
    justify-content: space-between;
    line-height: 18px;
    margin-bottom: 16px;
    color: $text-color-another;
    
    &-value {
      // write styles here
    }
    
    &-header {
      // write styles here
    }
  }
```